ci/launch_cluster.sh

```diff
@@ -33,24 +48,35 @@ wget 'https://downloads.dcos.io/dcos-test-utils/bin/linux/dcos-launch' && chmod
 envsubst <<EOF > config.yaml
 ---
 launch_config_version: 1
-template_url: $TEMPLATE
 deployment_name: $DEPLOYMENT_NAME
-provider: aws
+platform: aws
+instance_type: m4.large
+os_name: cent-os-7-dcos-prereqs
 aws_region: us-west-2
+provider: onprem
+installer_url: $INSTALLER_URL
 key_helper: true
-template_parameters:
-    DefaultInstanceType: m4.large
-    AdminLocation: 0.0.0.0/0
-    PublicSlaveInstanceCount: 1
-    SlaveInstanceCount: 3
-    LicenseKey: $DCOS_LICENSE
+num_masters: 3
+fault_domain_helper:
+  LocalZone:
+    num_zones: 2
+    num_public_agents: 1
+    num_private_agents: 2
+    local: true
+  RemoteZone:
+    num_zones: 2
+    num_private_agents: 2
+dcos_config:
+  cluster_name: Marathon $DEPLOYMENT_NAME
+  exhibitor_storage_backend: zookeeper
+  exhibitor_zk_path: /exhibitor
+  resolvers:
+    - 8.8.8.8
+    - 8.8.4.4
+  master_discovery: static
+$EXTRA_DCOS_CONFIG
 EOF
 
-if [[ -z `curl $TEMPLATE | grep 'LicenseKey'` ]]; then
-    sed -i '/LicenseKey/d' 'config.yaml'
-    echo 'Removed LicenseKey parameter as it was not in template'
-fi
-
 ./dcos-launch create
 if [ $? -ne 0 ]; then
```

Comment:
According to shellcheck we could do `if [ ./dcos-launch create -ne 0 ]; then`. I'm fine with not cleaning up though.

Reply:
Since we are doing something similar with the wait below, let's change it too.